site/docs/configuration.md

```diff
@@ -78,6 +78,12 @@ Iceberg tables support table properties to configure table behavior, like the de
 | --------------------------------------------- | -------- | ------------------------------------------------------------- |
 | compatibility.snapshot-id-inheritance.enabled | false    | Enables committing snapshots without explicit snapshot IDs    |
 
+### Format version
```

Comment:
I think it should be under the table properties section, and a part of the "reserved table properties". See the javadoc for more details, but the intention is that there might be other reserved properties in the future.

Reply:
ok. updated.